chirp/drivers/ft1d.py

```diff
@@ -1089,6 +1104,21 @@ def _debank(self, mem):
         for bank in bm.get_memory_mappings(mem):
             bm.remove_memory_from_mapping(mem, bank)
 
+    def enforce_band(self, mem, freq, num, regtype, flag):
```

Comment:
I don't see where you're ever calling this. Am I missing it? If I try this whole PR locally and I try to set 144MHz into the 50MHz home channel, it does not raise an error and changes to 500MHz.

The right way to do this is override `validate_memory()` in your class and return error messages if/when necessary, which the UI will show to the user.